libraries/alicloud_ecs_instance.rb

```diff
@@ -76,7 +76,7 @@ def fetch_instance(opts)
         action: "DescribeInstances",
         params: {
           'RegionId': opts[:region],
-          'InstanceIds': [opts[:instance_id]],
+          'InstanceIds': "[\"#{opts[:instance_id]}\"]",
```

Comment:
```suggestion
          'InstanceIds': [opts[:instance_id]].to_json,
```

Reply:
as per [docs](https://www.alibabacloud.com/help/doc-detail/25506.htm#:~:text=The%20IDs%20of%20instances.), it is expected to be a JSON array.. so the native way to build a JSON array is using `to_json` on the array object.